libs/blocks/merch-card/merch-card.js

```diff
@@ -228,6 +229,23 @@ const simplifyHrs = (el) => {
   });
 };
 
+function createQuantitySelect(el) {
+  const quantitySelectConfig = el.querySelector('ul');
+  if (!quantitySelectConfig) return null;
+  const configMarkup = quantitySelectConfig.querySelector('li');
+  if (!configMarkup || !configMarkup.textContent.includes('Quantity')) return null;
+  const config = configMarkup.querySelector('ul').querySelectorAll('li');
+  if (config.length !== 2) return null;
+  const attributes = {};
+  attributes.title = config[0].textContent.trim();
+  const quantityValues = config[1].textContent.split(',').map((value) => value.trim());
+  if (quantityValues.length !== 3) return null;
+  [attributes.min, attributes.max, attributes.step] = quantityValues.map(Number);
```

Comment:
please make sure that `quantityValues.map(Number);` is safe against to invalid values.
e.g:
`['', 'a', "0"].map(Number)`

the minimal value for any of the above is 1.